code/src/java/plugin/lsttokens/spell/CompsToken.java

```diff
@@ -95,7 +96,7 @@ protected ParseResult parseTokenWithSeparator(LoadContext context,
 		{
 			sb.append(Constants.LST_DOT_CLEAR);
 		}
-		if (added != null && !added.isEmpty())
+		if (!CollectionUtils.emptyIfNull(added).isEmpty())
```

Comment:
This seems questionable to me.  This trades a one-operation command (null check) for a method call, returning an object, and then checking it's empty.  I also question whether this is actually clearer...